frontend-react/src/hooks/network/ServiceHooks.ts

```diff
@@ -0,0 +1,101 @@
+import {
+    Dispatch,
+    SetStateAction,
+    useCallback,
+    useEffect,
+    useMemo,
+    useState,
+} from "react";
+import { AccessToken } from "@okta/okta-auth-js";
+
+import { auxExports } from "../UseCreateFetch";
+import {
+    MembershipState,
+    MemberType,
+    ServiceSettings,
+} from "../UseOktaMemberships";
+import { RSService, servicesEndpoints } from "../../config/endpoints/settings";
+import { RSNetworkError } from "../../utils/RSNetworkError";
+
+type ServiceState = ServiceSettings & {
+    setActiveService: Dispatch<SetStateAction<string>>;
+};
+/** Fetches membership services (senders, receivers) from the ReportStream API. Updates
+ * whenever the given {@link MembershipState} or AccessToken (Okta) are updated.
+ * @remarks ONLY FOR USE WITHIN {@link SessionContext} */
+export const useMemberServices = (
+    state: MembershipState,
+    token: AccessToken | undefined
+): ServiceState => {
+    const [senders, setSenders] = useState<RSService[] | undefined>();
+    const [receivers, setReceivers] = useState<RSService[] | undefined>();
+    const [activeService, setActiveService] = useState<string>("default");
```

Comment:
Defaulting to `"default"` as _so much_ of the site relies on it. This will change when we actually hook up features to this. Daily Data is unbroken by this, and Validate has been modified to read the value from here, as it was already a consumer of the `activeMembership.service` value.